### ISSUE TITLE
swe_houses_ex2 is broken on web

### ISSUE THREAD

**Issue by timashev** [vm75/sweph.dart#10]
This code:
```dart
print('${Sweph.swe_houses_ex2(
  Sweph.swe_julday(2000, 1, 1, 12, CalendarType.SE_GREG_CAL),
  SwephFlag.SEFLG_TROPICAL,
  30,
  60,
  Hsys.B
).cusps}');
```
prints in Chrome:
```
[
  8.826483935527076e-308,
  0,
  2.122124588e-314,
  132.80437141021736,
  158.84942420283255,
  196.56974623186812,
  232.48806967815233,
  265.3125030429241,
  288.54205505717465,
  312.80437141021736,
  338.8494242028326,
  16.569746231868116,
  52.48806967815234
]
```
instead of, e.g. correct output on macOS:
```
[
  0.0,
  85.31250304292412,
  108.54205505717465,
  132.80437141021736,
  158.84942420283255,
  196.56974623186812,
  232.48806967815233,
  265.3125030429241,
  288.54205505717465,
  312.80437141021736,
  338.8494242028326,
  16.569746231868116,
  52.48806967815234
]
```
So first 3 values on the web are corrupted (probably it is just a coincidence; if it is a memory corruption, then it could have another effects in other apps).

Probably other functions are affected as well, I only use these functions in my code at the moment:
* Sweph.swe_julday
* Sweph.swe_set_ephe_path
* Sweph.swe_calc_ut
* Sweph.swe_houses_ex2
* Sweph.swe_fixstar2_ut
* Sweph.swe_split_deg

and I noticed the only problem on the web with `cusps` in `Sweph.swe_houses_ex2` results.

**Comment by vm75** [comment]
Thanks for reporting this @timashev. I will debug and see what is happening.

**Comment by vm75** [comment]
Fixed in version 2.10.3+14
